oracle-lib/test/cmr/oracle/sql_utils_test.clj

```diff
@@ -0,0 +1,5 @@
+(ns cmr.oracle.sql-utils-test
+  (:require
+   [clojure.test :refer :all]))
+
+(deftest ^:kaocha/pending utils-placeholder-test)
```

Comment:
Similar to the comment on the spatial test - maybe have a comment about why kaocha requires placeholder tests